lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -302,13 +297,7 @@ func (h *StorageHandlersImpl) VolumeStoresList(params storage.VolumeStoresListPa
 			})
 	}
 
-	resp := &models.VolumeStoresListResponse{
-		Stores: make(map[string]string),
-	}
-
-	for name, ds := range stores {
-		resp.Stores[name] = ds.String()
-	}
+	resp := &models.VolumeStoresListResponse{Stores: stores}
```

Comment:
Is the default behavior here that the respond will call the .String() function on the values here? since stores is of type map[string]Url